nf_core/pipeline-template/nextflow.config

```diff
@@ -204,12 +204,22 @@ profiles {
 }
 
 {% if nf_core_configs -%}
-// Load nf-core custom profiles from different Institutions
-includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
+// Load nf-core custom profiles from different institutions
+// Rationale:
+// - The new config parser does not allow for try, if or def on a top-level, so the old try notation needed updating.
+// - Normally, we include the nf-core custom config from GitHub, the custom_config_base defaults to the nf-core/configs repository.
+// - If a execution environment does not have access to the internet, the pipeline run would, however, fail because the custom config is not available.
+// - Therefore, we do not want to include the custom config from GitHub if the NXF_OFFLINE environment variable is set, yet allow for a local path.
+// - The institutional configs should be included even in offline mode, if they are locally available.
```

Comment:
```suggestion
```
This is a nice explanation for the PR description but I don't think we need to have it in the template :)